substrate/frame/assets/src/functions.rs

```diff
@@ -228,20 +237,21 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		let account = Account::<T, I>::get(&id, who).ok_or(Error::<T, I>::NoAccount)?;
 		ensure!(!account.status.is_frozen(), Error::<T, I>::Frozen);
 
-		let amount = if let Some(frozen) = T::Freezer::frozen_balance(id, who) {
-			// Frozen balance: account CANNOT be deleted
```

Comment:
IMO no hehe

Precisely, because the new model doesn't consider `frozen`, and `held` balances as separate components, but calculates them as a whole, and considers for `unspendable`/`untouchable` and `spendable`/`reducible` balances.